README.md

```diff
@@ -37,20 +37,6 @@ Kiwix-Android is a multi-module project, in 99% of scenarios you will want to bu
 If you are interested in our custom apps they have their own repo [kiwix-android-custom](https://github.com/kiwix/kiwix-android-custom
 ) that goes into further detail
 
-## Libraries Used
```

Comment:
I don't think we want to delete our entire library section

Reply:
I thought if someone wants to check what are the libraries used he can check the gradle files.
This was just to make our readme concise.

Reply:
Aye it is a bit of a nuisance to maintain. Maybe lets have this discussion on another ticket though, a different concern